src/constants/Tags.js

```diff
@@ -9,16 +9,21 @@
 */
 
 import { TopicTypes } from 'docc-render/constants/TopicTypes';
+import { ChangeNames } from 'docc-render/constants/Changes';
 
-export const HIDE_DEPRECATED = 'navigator.tags.hide-deprecated';
-
-export const FILTER_TAGS = {
+export const TYPE_TAGS = {
```

Comment:
We should rename the values of `TOPIC_TYPE_TO_TAG` to use `TYPE_TAGS` instead of `FILTER_TAGS` too:

```js
export const TOPIC_TYPE_TO_TAG = {
  [TopicTypes.article]: TYPE_TAGS.articles,
  [TopicTypes.learn]: TYPE_TAGS.tutorials,
  [TopicTypes.overview]: TYPE_TAGS.tutorials,
  [TopicTypes.resources]: TYPE_TAGS.tutorials,
  [TopicTypes.sampleCode]: TYPE_TAGS.sampleCode,
  [TopicTypes.section]: TYPE_TAGS.tutorials,
  [TopicTypes.tutorial]: TYPE_TAGS.tutorials,
  [TopicTypes.project]: TYPE_TAGS.tutorials,
  [TopicTypes.httpRequest]: TYPE_TAGS.webServiceEndpoints,
};
```

Reply:
I think this is not needed. Because `TYPE_TAGS` is just mean to be a subset of all available tags. It's probably better to use the variable that contains all "possible tags". 
What do you think?

Reply:
Yes, it won't change the functionality but it just makes clear that the `TOPIC_TYPE_TO_TAG` depends on the specific `TYPE_TAGS` and not other tags from `FILTER_TAGS` like API Changes for example

Reply:
Anyways, it's not a blocker, so I'm approving the PR :)